frontend/src/pages/adminRoles.js

```diff
@@ -189,7 +192,9 @@ class AdminRoles extends React.Component {
             <Button
               color="warning"
               size="sm"
-              onClick={() => this.setState({ showPasswordModal: false })}
+              onClick={() =>
+                this.setState({ showPasswordModal: false, newRole: -1, selectedUser: -1 })
```

Comment:
nit: what are the -1 for

Reply:
mostly to show the value is supposed to be an integer and it's unassigned to a user index in the initial state